mk/travis-install-kcov.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2016 Pietro Monteiro
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+set -ex
+
+curl -L https://github.com/SimonKagstrom/kcov/archive/master.tar.gz | tar -zxf -
```

Comment:
Is it necessary to use the trunk, or can we use the latest release linked of [the kcov home page](http://simonkagstrom.github.io/kcov/index.html), which is https://github.com/SimonKagstrom/kcov/archive/v30.tar.gz?

If we can't use the release version, please add a comment explaining why, so that we can switch to a release version if/when those issues are resolved.

Also, if we can't use the release version, should we instead use a specific revision of kcov?

If we just always use the trunk of kcov, then we're at risk of our code coverage breaking whenever the kcov trunk breaks, and I'd like to avoid that.